tests/test_events.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+import warnings
+from datetime import datetime
+
+import rasa_sdk.events as events
+
+
+def test_reminder_scheduled_correctly():
+    with warnings.catch_warnings(record=True) as warning:
```

Comment:
can we please test the warnings as described here https://docs.pytest.org/en/latest/warnings.html#warns (`with pytest.warns(FutureWarning) ...`)